src/packages/recompose/__tests__/pick-test.js

```diff
@@ -0,0 +1,29 @@
+import test from 'ava'
+import pick from '../utils/pick'
+
+test('pick', t => {
+  t.deepEqual(
+    pick({
+      one: '1',
+      dont: 'include',
+      two: {
+        deep: {
+          value: 'true'
+        }
+      },
+      root: {},
+      three: '3'
+    }, [
+      'one',
+      ['two', 'deep', 'value'],
+      ['root', 'missed'],
```

Comment:
If we try to go deeper `['root', 'missed', 'something']` would still return `undefined`.